cmd/wavelet/cli.go

```diff
@@ -141,6 +142,62 @@ func NewCLI(client *skademlia.Client, ledger *wavelet.Ledger, keys *skademlia.Ke
 			Aliases: []string{"quit", ":q"},
 			Action:  a(c.exit),
 		},
+		{
+			Name:   "update-parameters",
+			Action: a(c.updateParameters),
+			Flags: []cli.Flag{
+				cli.IntFlag{
+					Name:  "snowballK",
```

Comment:
Rather than camel-case, best have them dotted-paths. For example, `snowballK` is `snowball.k`.